plugins/pulp_rpm/plugins/migrations/0042_add_suse_fields_to_errata.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from pulp.server.db.connection import get_collection
+
+
+_logger = logging.getLogger('pulp_rpm.plugins.migrations.0042')
+
+
+def migrate(*args, **kwargs):
+
+    erratum_collection = get_collection('units_erratum')
+
+    for errata in erratum_collection.find({}):
```

Comment:
For users it's more clear what's happening when they see the progress of the migration. Consider adding progress like [here](https://github.com/werwty/pulp_rpm/blob/1bd728d13e549586b96de6d04fa60cb4ee43c726/plugins/pulp_rpm/plugins/migrations/0040_errata_pkglist_collection.py#L36)